crates/red_knot_python_semantic/resources/mdtest/import/errors.md

```diff
@@ -55,3 +55,27 @@ from b import x
 
 x = "foo"  # error: [invalid-assignment] "Object of type `Literal["foo"]"
 ```
+
+## Import cycle
+
+```py path=a.py
+class A:
+    pass
```

Comment:
Style micro-nit: we generally use `...` rather than `pass` in empty class bodies in mdtests, because blacken-docs (an autoformatter we run on our mdtests in CI) formats the classes more concisely if you do that